big_scape/cli/cli_common_options.py

```diff
@@ -271,7 +271,8 @@ def common_cluster_query(fn):
         ),
         click.option(
             "--classify",
-            type=click.Choice(["class", "category"]),
+            type=click.Choice(["none", "class", "category"]),
+            default="class",
```

Comment:
in query mode we dont want a default class, we want default mix, so this should probs be moved to the individual cluster and query options